code/api/src/seeders/1-user.js

```diff
@@ -20,6 +26,9 @@ module.exports = {
         email: 'user@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.user,
+        image: 'https://pbs.twimg.com/profile_images/1237550450/mstom_400x400.jpg',
+        bio: 'I am Tom from MySpace and I\'ve got style',
+        shippingAddress: JSON.stringify(jsonAddress),
```

Comment:
This makes sooo much sense. Looking forward to walking through this later!! LGTMMMMM

Reply:
Nice, Tom from Myspace